packages/launcher-client/src/client/impl/mock.launcher.client.ts

```diff
@@ -122,6 +124,10 @@ export default class MockLauncherClient implements LauncherClient {
     }, 2500);
   }
 
+  public async gitProviders(): Promise<GitProvider[]> {
+    return gitProviders;
```

Comment:
This line of code seems to fix the problem and work:
   `return gitProviders as GitProvider[];`

Reply:
right, but it shouldn't need that right? But let's do that anyway